.github/workflows/validatePR.yml

```diff
@@ -0,0 +1,23 @@
+name: validate pr
+on:
+  pull_request:
+    branches: [develop]
+
+jobs:
+  pr-validation:
+    uses: salesforcecli/github-workflows/.github/workflows/validatePR.yml@main
+  code-quality: 
+    strategy:
+      matrix:
+        node_version: [lts/-1]
+      fail-fast: false
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node_version }}
+          cache: npm
+      - run: npm ci
+      - name: Lint
+        run: npm run lint
```

Comment:
validate using the shared workflow + a lint step

Reply:
Love this addition - fwiw if you don't intend to use multiple node versions, I don't think you need to include that strategy section (especially since you've set `fail-fast:false`)

Reply:
Yeah good call...I originally had it running on lts/* and lts/-1 but in the end decided it wasn't worth doing the lint on multiple node version but left the setup out of laziness.  Think I should remove that strategy setup?